bin/nvcc_wrapper

```diff
@@ -90,6 +90,11 @@ replace_pragma_ident=0
 # Mark first host compiler argument
 first_xcompiler_arg=1
 
+# Allow for setting temp dir without setting TMPDIR in parent (see https://docs.olcf.ornl.gov/systems/summit_user_guide.html#setting-tmpdir-causes-jsm-jsrun-errors-job-state-flip-flop)
+if [[ ! -z ${_NVCC_WRAPPER_TMPDIR_OVERRIDE+x} ]]
+then
+  export TMPDIR=${_NVCC_WRAPPER_TMPDIR_OVERRIDE}
```

Comment:
I don't think you need the `export` here
```suggestion
  TMPDIR=${_NVCC_WRAPPER_TMPDIR_OVERRIDE}
```

Reply:
Probably not, if it's already set in the environment.  But since we can't guarantee that, I think it's needed.